test/azlinux_test.go

```diff
@@ -478,16 +484,29 @@ ListenStream=127.0.0.1:8080
 WantedBy=sockets.target
 								`,
 								},
+								"foo.conf": {
+									Contents: `
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
+								`,
+								},
 							},
 						},
 					},
 				},
 			},
 			Artifacts: dalec.Artifacts{
-				SystemdUnits: map[string]dalec.SystemdUnitConfig{
-					"src/foo.service": {},
-					"src/foo.socket": {
-						Enable: true,
+				Systemd: &dalec.SystemdConfiguration{
+					Units: map[string]dalec.SystemdUnitConfig{
+						"src/foo.service": {},
+						"src/foo.socket": {
+							Enable: true,
+						},
+					},
+					Dropins: map[string]dalec.SystemdDropinConfig{
+						"src/foo.conf": {
```

Comment:
Can we add another test config file here that is specifically for the `foo.socket` unit?

Reply:
done.